Build/libHttpClient.142.XDK.C/libHttpClient.142.XDK.C.vcxproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ApplicationEnvironment>title</ApplicationEnvironment>
-    <ProjectGuid>{9E0EB8C2-40CD-448F-9B98-DAF9830EF9AD}</ProjectGuid>
+    <ProjectGuid>{57C681F7-280F-45B8-9FA9-D556C24F2053}</ProjectGuid>
```

Comment:
guids changing is not a good sign, have the templates gone out of sync again? are these guids copy pasted from somewhere else?

Reply:
agreed.  template checked in should be keeping it unchanged:

C:\git\forks\libHttpClient>findstr /sip 9E0EB8C2 *.vcxproj
Utilities\CMake\template-libHttpClient.142.XDK.C.vcxproj:    <ProjectGuid>{9E0EB8C2-40CD-448F-9B98-DAF9830EF9AD}</Projec

---
In reply to: [385430849](https://github.com/microsoft/libHttpClient/pull/495#discussion_r385430849) [](ancestors = 385430849)